modules/distributed-tracing-rn-technology-preview.adoc

```diff
@@ -0,0 +1,27 @@
+////
+Module included in the following assemblies:
+- rhbjaeger-release-notes.adoc
+////
+
+[id="distributed-tracing-rn-tech-preview_{context}"]
+= Technology Preview
+////
+Provide the following info for each issue if possible:
+Description -  Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.  Avoid the word “supports” as in [product] now supports [feature] to avoid customer confusion with full support.  Say, for example, “available as a Technology Preview.”
+Package - A brief description of what the customer has to install or enable to use the Technology Preview feature.    (e.g., available in quickstart.zip on customer portal, JDF website, container on registry, enable option, etc.)
+////
+
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+== {ProductName} 2.0.0 Technology Preview 1 -- Retain only for 4.6-4.8
+When you install the Jaeger Operator you can select a Technology Preview version of Jaeger.  This gives you access to clients and infrastructure for exporting tracing data to Jaeger based on the link:https://opentelemetry.io/[OpenTelemetry framework].  Note that this version is not supported for production environments.
+
+The OpenTelemetry collector allows developers to instrument their code with vendor agnostic APIs, avoiding vendor lock-in and opening the door to a growing ecosystem of observability tooling.
+
+== Red Hat OpenShift distributed tracing data collection
+
+Red Hat OpenShift distributed tracing data collection, based on OpenTelemetry 0.33, is technology preview. This includes operator and collector capabilities.
```

Comment:
The OpenTelemetry collector has some capabilities beyond just tracing, we may want to highlight that only the tracing parts should be used at this point.

Perhaps something like:

"Red Hat OpenShift distributed tracing data collection is based on OpenTelemetry's operator and collector. The collector can be used to receive traces in either the OpenTelemetry or Jaeger protocol and send the trace data to the OpenShift distributed tracing platform. Other capabilities of the collector are not supported at this time."

@pavolloffay any thoughts on this?

Reply:
> The OpenTelemetry collector has some capabilities beyond just tracing, we may want to highlight that only the tracing parts should be used at this point.

agree, we don't want people to prevent of using them, but we should mention this is not supported in our product